elftools/dwarf/structs.py

```diff
@@ -226,6 +226,11 @@ def _create_lineprog_header(self):
             self.Dwarf_uint16('version'),
             self.Dwarf_offset('header_length'),
             self.Dwarf_uint8('minimum_instruction_length'),
+			
+			If(lambda ctx: ctx['version'] >= 4, 
+				self.Dwarf_uint8("maximum_operations_per_instruction"),
+				1),
```

Comment:
The If macro encapsulates the 1 in a Value object and so this behaves correctly.

From macros.py:
```python
def If(predicate, subcon, elsevalue = None):
    return IfThenElse(subcon.name,
        predicate,
        subcon,
        Value("elsevalue", lambda ctx: elsevalue)
    )
```